tests/misc/BuildManPageTest.py

```diff
@@ -75,8 +75,8 @@ def test_mk_options(self):
     def test_mk_footer(self):
         uut = ManPageFormatter(app_name, ext_sections=sections)
         self.assertEqual(uut._mk_footer(),
-                         ".SH {0}\n"
-                         " {1}".format(section_name.upper(), section_text))
+                         ".SH {}\n {}".format(
```

Comment:
@Makman2 What's up here? I vaguely remember, we were accepting `{0} {1}` instead of `{ } { }` style.
 :D